filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -254,6 +295,20 @@ DECL_DRIVER_API_TAGGED_R_N(backend::TextureHandle, importTexture,
         uint32_t, depth,
         backend::TextureUsage, usage)
 
+DECL_DRIVER_API_TAGGED_R_N(backend::TextureHandle, importTextureAsync,
```

Comment:
Just to be sure I understand, this actually wouldn't need to be async (not doing anything expensive), but we have async version so that it can be queued with the async texture creation, correct?